doc/aggregation.md

```diff
@@ -0,0 +1,164 @@
+# Enable ClickHouse aggregation
+
+It's enabled by default since [#157](https://github.com/lomik/graphite-clickhouse/pull/157). You can disable it to use aggregation in graphite-clickhouse.
+
+```
+[clickhouse]
+# ClickHouse-side aggregation
+internal-aggregation = true
+# maximum number of points per metric. It should be set to 4096 or less for ClickHouse older than 20.8
+# https://github.com/ClickHouse/ClickHouse/commit/d7871f3976e4f066c6efb419db22725f476fd9fa
+max-data-points = 1048576
+```
+
+The only known _frontend_ supporting passing `maxDataPoints` from requests is [carbonapi>=0.14](https://github.com/go-graphite/carbonapi/releases/tag/0.14.0). The used protocol should be `carbonapi_v3_pb`, see [config](https://github.com/go-graphite/carbonapi/blob/main/doc/configuration.md#upstreams)->backendv2->backends->protocol.
+
+But even without the mentioned adjustments, `internal-aggregation` improves the whole picture by implementing whisper-like archives (see below).
+
+## Compatible ClickHouse versions
+The feature uses ClickHouse aggregation combinator [-Resample](https://clickhouse.tech/docs/en/sql-reference/aggregate-functions/combinators/#agg-functions-combinator-resample). This aggregator is available since version [19.11](https://github.com/ClickHouse/ClickHouse/commit/57db1fac5990a7227e720c9dd438d88a381d298f)
+
+*Note*: version 0.12 is compatible only with CH 20.1.13.105, 20.3.10.75, 20.4.5.36, 20.5.2.7 and greater since it uses -OrNull modifier.
```

Comment:
`And greater` - `or newer`. This is also applicable for the next session.

In general might be a good idea to create a small table with version compatible matrix and tell what is the minimum recommended version (in general).

Reply:
I'd simplify it to "use the latest LTS" =)